scripts/linux/trcE2ETest.sh

```diff
@@ -870,9 +872,10 @@ function configure_connectivity_settings() {
     NETWORK_CONTROLLER_RUNPROFILE=${NETWORK_CONTROLLER_RUNPROFILE:-Offline}
     TEST_DURATION="${TEST_DURATION:-01:00:00}"
 
+    VERIFICATION_DELAY="${VERIFICATION_DELAY:-00:15:00}"
```

Comment:
I think we are asking ourselves for trouble here. Previously, the eventHub has 45 minutes to ingress and the messages before TRC create a report. Now it only has 15 minutes after the test completion to ingress.... This screams instability.

Reply:
This is reverting the verification delay to what is was before my previous PRs changes, which had no effect on real verification delay.